web.browser/src/components/Week/index.js

```diff
@@ -0,0 +1,39 @@
+import React, { PropTypes } from 'react';
+import slug from 'slug';
+import { Link } from 'react-router';
+import { List, ListItem } from 'material-ui/List';
+import Subheader from 'material-ui/Subheader';
+import Divider from 'material-ui/Divider';
+
+const Week = ({ week, categories, filterClick }) => {
+  return (
+    <div>
+      <List>
+        <Subheader>{week}</Subheader>
+        {categories.map((category, i) => (
+          <div key={i.toString()}>
+            <Link
+              to={`/posts/${slug(category, { lower: true })}`}
+              style={{ textDecoration: 'none' }}
+            >
+              <ListItem
+                primaryText={category}
+                onClick={() => filterClick(category)}
+                />
```

Comment:
With only two attributes per element, it would probably be cleaner to put these as one-liners. It's currently pretty tough to read. 

You could also put the generation of these elements in either a function or in a component, so you `map` call would be way cleaner.

```jsx
import MyMenuComponent from './wherever';

...

{ categories.map((category, i) => <MyMenuComponent category={category} />

```